src/compiler/types.ts

```diff
@@ -892,41 +892,33 @@ module ts {
         referencedFiles: FileReference[];
 
         hasNoDefaultLib: boolean;
-        externalModuleIndicator: Node; // The first node that causes this file to be an external module
+
+        // The first node that causes this file to be an external module
+        externalModuleIndicator: Node;
         languageVersion: ScriptTarget;
         identifiers: Map<string>;
         
-        // @internal
-        nodeCount: number;
-
-        // @internal
-        identifierCount: number;
-
-        // @internal
-        symbolCount: number;
+        /* @internal */ nodeCount: number;
+        /* @internal */ identifierCount: number;
+        /* @internal */ symbolCount: number;

```

Comment:
@ahejlsberg Hey Anders, 'internal' is used to prevent members/symbols/etc. from appearing in our public d.ts files for the compiler/LS API.  We are using them so that we can expose things like 'SourceFile' without necessarily having to expose all their innards.  Useful places for 'internal' include:

1) lazily initialized fields.  We don't want to expose these fields directly as users could see them in an inconsistent state.  Instead, there are helper functions for getting the data that can then check and set the values.

2) internal implementation details.  While we may have some functions that mutate the world, we don't necessarily want consumers seeing or accessing these.  THey're just for our compiler to call itself, but not really something that is part of our 'public' API.